cypress/integration/sdvv-frontend_spec.js

```diff
@@ -0,0 +1,48 @@
+describe('Build - Smoke Test', () => {
+
+  const links = [
+    { text: 'FAQ', url: '/faq' },
+    { text: 'About', url: '/about' },
+  ]
+  
+  links.forEach( link => {
+    it(`Clicking ${link.text} navigates to new URL`, () => {
+      cy.visit('')
+      cy.contains(link.text).click()
+      cy.url().should('include', link.url)
+    })
+  })
+
+  //Don't open link, just verify that it is set up correctly
+  it('Verify Register to Vote Link', () => {
+    cy.visit('')
+    cy.contains('Register to Vote')
+      .should('have.attr', 'href')
+      .and('include', 'registertovote.ca.gov')
+  })
+
+  const candidates = [
+    { text: 'Mayor', url: '/mayor' },
+    { text: 'City Attorney', url: '/city-attorney' },
+    { text: 'City Council', url: '/city-council' },
+  ]
+
+  candidates.forEach( candidate => {
```

Comment:
Great job with these! 
This is actually the same as the test on line 8. Not necessary, but they could be combined.

Reply:
Change made.